diff --git a/node_modules/react-native-draggable/Draggable.tsx b/node_modules/react-native-draggable/Draggable.tsx
index 363ab4a..e588216 100644
--- a/node_modules/react-native-draggable/Draggable.tsx
+++ b/node_modules/react-native-draggable/Draggable.tsx
@@ -76,6 +76,7 @@ export default function Draggable(props: IProps) {
     onPressIn,
     onPressOut,
     onRelease,
+    onReverse, // Patched by Beau. See https://github.com/tongyy/react-native-draggable/issues/41#issuecomment-789320290
     x,
     y,
     z,
@@ -113,13 +114,29 @@ export default function Draggable(props: IProps) {
     },
     [disabled],
   );
-
+/////////////////////////////////////////////////////////////////////  
+/*
   const reversePosition = React.useCallback(() => {
     Animated.spring(pan.current, {
       toValue: {x: 0, y: 0},
       useNativeDriver: false,
     }).start();
   }, [pan]);
+*/
+
+
+  const reversePosition = React.useCallback(() => {
+/*    Animated.spring(pan.current, {
+      toValue: {x: 0, y: 0},
+      useNativeDriver: false,
+    }).start();
+      */
+
+      pan.current.setValue( {x: 0, y: 0});
+  }, [pan]);
+
+///////////////////////////////////////////////////////////////////
+
 
   const onPanResponderRelease = React.useCallback(
     (e: GestureResponderEvent, gestureState: PanResponderGestureState) => {
